### ISSUE TITLE
Expand predicate filters

### ISSUE THREAD

**Issue by pmkc** [GoogleCloudPlatform/spark-bigquery-connector#4]
This is blocked on the API limits at the moment.

**Comment by darrenhaken** [comment]
Any idea for the timeline around this? Appetite for contributions?

**Comment by kmjung** [comment]
The issue is on the server side, unfortunately, not in the connector. We're working on expanding push-down filtering further at the moment, but I don't have a specific ETA for you.

**Comment by darrenhaken** [comment]
Ah that’s a shame. I guess the same applies for the date predicate GitHub issue too?

**Comment by kmjung** [comment]
The date predicate issue may be a separate issue. I will take a look at that one as soon as I can -- it should be a simple repro, in theory -- but haven't had time to do so yet.

**Comment by kmjung** [comment]
This was resolved by https://github.com/GoogleCloudPlatform/spark-bigquery-connector/pull/39.
